addon/components/vertical-collection/component.js

```diff
@@ -197,6 +197,35 @@ const VerticalCollection = Component.extend({
     }
   },
 
+  /* Public API Methods 
+     @index => number
+     This will return offset height of the indexed item.
+  */
+  offsetForIndex(index) {
+    const { _radar } = this;
+    let scrollTop = _radar.getOffsetForIndex(index);
+    _radar._scrollTop = scrollTop;
+    _radar.update();
+    return new Promise(function(resolve, reject) {
+      run.later(() => {
+        resolve(_radar.getOffsetForIndex(index));
+      });
+    });
+  },
+
+   /* Public API Methods 
+     @index => number
+     This will return true or false based on the indexed item in the scrollcontainer viewport
+  */
+  checkIfIndexIsInViewport(index) {
+    const { _radar } = this;
+    if (index >= _radar._firstItemIndex && index <= _radar._lastItemIndex) {
```

Comment:
I am not quite sure what are we trying to check here

Reply:
We are checking if the particular item is in viewport or not.